_sass/method.scss

```diff
@@ -134,32 +134,37 @@
   .method--panel--front {
     position: relative;
 
-footer {position: relative;  width: 100%;
-display:table;margin-top: 3rem;}
-
-  @media screen and (min-width: 800px) {
     footer {
-      position: absolute;
+      position: relative;
       width: 100%;
-      bottom: 0;
-      left: 0;
-      padding: 0 40px 20px;
       display: table;
+      margin-top: 3rem;
+    }
+
+    @media screen and (min-width: 800px) {
+      footer {
+        position: absolute;
+        width: 100%;
+        bottom: 0;
+        left: 0;
+        padding: 0 40px 20px;
+        display: table;
+      }
     }
   }
-}
 
 @media screen and (min-width: 800px) {
   .method--panel--front {
     position: relative;
-    padding-bottom:15rem;
+    padding-bottom: 15rem;
   }
 }
 
   // method--panel--back
   .method--panel--back {
   	font-size: $small-font-size;
   	background: $color-gray-lightest;
+    border-radius: 0 3px 3px 0;
```

Comment:
This is great! I noticed at smaller screens the lower left of the card needs a rounded corner too. 

<img width="228" alt="screen shot 2017-08-25 at 11 09 50 am" src="https://user-images.githubusercontent.com/1449852/29719884-f6e3990c-8985-11e7-8eee-e42b213e0394.png">